doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -0,0 +1,136 @@
+.. _deploy-cephadm-mgmt-gateway:
+
+=============
+Mgmt Gateway
+=============
+
+Deploying mgmt-gateway
+======================
+
+Starting from version 19.x.x, the `mgmt-gateway` service introduces a new design for Ceph applications
```

Comment:
I think we usually name them when referencing this way
>In Ceph releases beginning with Squid,

Reply:
fixed